### DISCUSSION TITLE
El programador remoto en tiempo de COVID

### DISCUSSION THREAD

**Discussion by David-Lazaro-Fernandez** [gittogethers/octogatosconf22#75] [Talks in Spanish]
Ponente: **Jose Javier Columbie**

> En el camino para lograr un equilibrio entre el trabajo y la vida personal (que la mayoría dice no ser realista en software) nos dimos cuenta de que la mejor manera de hacer las cosas es soltar. Soltar el ego, soltar el perfeccionismo, soltar las expectativas ¿Suena loco verdad? 

> Bueno, al explorar los hábitos de desarrolladores remotos reconocidos, encontramos algunos patrones y preguntas realmente interesantes. 

> Por ejemplo: Fallar te hace pensar de una manera que el éxito no puede. 

> El software no se trata de código, se trata de valor. ¿Cómo prosperamos con la abrumadora cantidad de información que procesamos todos los días? ¿Cómo logramos claridad y evaluamos nuestras prioridades? ¿Es lo mismo trabajo remoto que remoto en COVID? Profundicemos.

**Comment by jsiesquen** [comment]
Hay leyes que ultimamente respalden el trabajo remoto?
